yewprint-doc/src/tag/mod.rs

```diff
@@ -172,8 +172,10 @@ crate::build_example_prop_component! {
                             label="Right icon"
                         />
                         <p>{"Select intent:"}</p>
+                        <ButtonGroup
+                            vertical=true
+                        >
```

Comment:
```suggestion
                        <ButtonGroup vertical=true>
```
If it doesn't reach the 100 chars limit it can be onelined